dataclients/kubernetes/kube_test.go

```diff
@@ -855,10 +855,10 @@ func TestHealthcheckInitial(t *testing.T) {
 
 		r, err := dc.LoadAll()
 		if err != nil {
```

Comment:
i think here we want to test if we actually have an error, so i recommend something like this:

```
if err == nil {
    t.Error("error expected")
}
```

and the call to `checkHealthCheck` is unnecessary from that point